plugins/aver-hid/fu-aver-safeisp-device.c

```diff
@@ -324,10 +324,7 @@ fu_aver_safeisp_device_write_firmware(FuDevice *device,
 	/* CX3 fw file size should be less than 256KB */
 	cx3_fw_buf = g_bytes_get_data(cx3_fw, &cx3_fw_size);
 	if (cx3_fw_size > 256 * 1024) {
-		g_set_error(error,
-			    G_IO_ERROR,
-			    G_IO_ERROR_INVALID_DATA,
-			    "cx3 file size is invalid");
+		g_set_error(error, G_IO_ERROR, G_IO_ERROR_INVALID_DATA, "cx3 file size is invalid");
```

Comment:
if you don't have any format parameters you should use g_set_error_literal instead -- but in this case it would be useful to print what the value *is* -- e.g. `"cx3 file size is invalid: 0x%x", (guint) cx3_fw_size);`